eng/Directory.Build.Common.props

```diff
@@ -138,11 +138,11 @@
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true' or '$(IsSamplesProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
     <IsPackable>false</IsPackable>
     <!-- List newest targets first so that recordings are made with latest, running tests from editor runs latest, etc. -->
-    <RequiredTargetFrameworks>net7.0;net6.0</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks>net8.0;net6.0</RequiredTargetFrameworks>
```

Comment:
What is the plan to remove net 6.0 given it is EOL?

Reply:
I guess it helps if I read the description.

Reply:
.NET 6 is officially done in Nov and .NET 9 is released then too, at that point this will then be changed to net9.0;net8.0